lib/std/math.zig

```diff
@@ -621,6 +621,30 @@ fn testDivFloor() void {
     testing.expect((divFloor(f32, -5.0, 3.0) catch unreachable) == -2.0);
 }
 
+pub fn divCeil(comptime T: type, numerator: T, denominator: T) !T {
+    @setRuntimeSafety(false);
+    if (numerator <= 0) return divTrunc(T, numerator, denominator);
+    if (@typeInfo(T) == .Float) {
+        if (denominator == 0) return error.DivisionByZero;
+        return @ceil(numerator / denominator);
+    }
+    return (try divFloor(T, numerator - 1, denominator)) + 1;
+}
+
+test "math.divCeil" {
+    testDivCeil();
+    comptime testDivCeil();
+}
+fn testDivCeil() void {
+    testing.expect((divCeil(i32, 5, 3) catch unreachable) == 2);
```

Comment:
I would recommend testing all of the sign combinations
```suggestion
    testing.expect((divCeil(i32, -5, -3) catch unreachable) == 2);
    testing.expect((divCeil(i32, 5, -3) catch unreachable) == 2);
```

Reply:
Fair point... drat I really thought it'd be this simple.